packages/xs-extra/xapi-nbd.master/opam

```diff
@@ -24,7 +24,7 @@ depends: [
   "sexplib"
   "mirage-block-unix"
   "io-page"
-  "mirage"
+  "mirage" {< "3.0.0"}
```

Comment:
Thanks for the suggestion, cleaned up the `opam` file, now it works with both `xs-opam` and the upstream repo: https://travis-ci.org/gaborigloi/xapi-nbd/builds/237973155